client/telemetry/src/lib.rs

```diff
@@ -428,34 +488,33 @@ enum Register {
 /// # let authority_id = 42_u64;
 /// # let set_id = (43_u64, 44_u64);
 /// # let authorities = vec![45_u64];
-/// telemetry!(CONSENSUS_INFO; "afg.authority_set";
-/// 	"authority_id" => authority_id.to_string(),
-/// 	"authority_set_id" => ?set_id,
-/// 	"authorities" => authorities,
+/// telemetry!(
+///     telemetry;
```

Comment:
It should mention that this is required to be an option. However, I'm not sure if we should maybe hide the option inside the option inside the handle.

Reply:
Fixed in 526d877ce

I think the Option thingy is just an implementation detail of the telemetry stuff. Maybe it's best to hide it away in a `TelemetryInner` object.

The only thing I like with the current implementation is that it makes it more obvious when looking at the type that this can be omitted entirely.